style/index.css

```diff
@@ -277,7 +277,7 @@ textarea {
     flex-shrink: 0;
 }
 
-.mentions__highlighter > strong {
+.mentions__highlighter > strong, .mentions__highlighter__caret > strong {
     background-color: seashell;
```

Comment:
What required this change and could this be reason for failing tests?

Reply:
See #122 where I investigated the backstop issue. Quoting the closing comment:

> The issue has been traced back to an update of Google Chrome. Chrome is used as our engine of choice in Backsop. A side effect of the update is that, for tests featuring the comment component's text box, MentionsInput, the text entry caret is placed at the beginning of the text box and not at the text's end as it used to be. This revealed a bug in the styling of the component, where the mention highlight would disappear if the mention came after the caret (reason: the text after the caret is wrapped in a CSS class which, unlike other `react-mentions`-specific classes, was left unstyled). So ultimately, it turned out not to be a regression, but the reveal of an existing bug.

Reply:
That took care of three failing tests. The other four failing tests were investigated in the comments above.